h/views/api/auth.py

```diff
@@ -224,14 +224,14 @@ def post(self):
 def debug_token(request):
     if not request.auth_token:
         raise OAuthTokenError(
-            "Bearer token is missing in Authorization HTTP header", "missing_token", 401
+            "Bearer token is missing in Authorization HTTP header", "missing_token"
```

Comment:
OAuthTokenError is always 401 (the only status code that was ever actually used) now.